README.md

```diff
@@ -19,7 +19,16 @@ composer require rezozero/subscribeme
 ```
 
 ```php
-$subscriber = \SubscribeMe\Factory::createFor('mailjet');
+use SubscribeMe\Factory;
+
+/**
+* We use Psr18 so you need to use a client that implements psr 18 like guzzle for example
+*/
+$factory = new Factory($client, $requestFactory, $streamFactory);
```

Comment:
How do we get `$client`, `$requestFactory`, `$streamFactory` variables?

You should add Guzzle example